legacy_tutorials/aer/9_pulse_simulator_from_backend.ipynb

```diff
@@ -109,22 +113,63 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "provider = IBMQ.load_account()\n",
-    "armonk_backend = provider.get_backend('ibmq_armonk')"
+    "armonk_backend = FakeArmonk()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Construct Rabi experiment schedules."
+    "Manual insertion of Hamiltonian parameters:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "# set the drive strength from a previously inferred value for the real Armonk device\n",
+    "getattr(armonk_backend.configuration(), 'hamiltonian')['vars']['omegad0'] = 63455017.85\n",
+    "\n",
+    "# set the qubit frequency from the estimate in the defaults\n",
+    "freq_est = armonk_backend.defaults().qubit_freq_est[0] / 10**9\n",
```

Comment:
Don't hardcode this based on defaults as this is a bug. Just hardcode the frequency eg. `5e9` or whatever it is. Also 10**9 involves an exponentiation. Better to use 1e9 which is directly an integer value in Python and will be more efficient.

Reply:
Thanks, will make these changes. (Especial thanks for the `1e9` tip, good to know.)